src/components/curation/ClinicalPhenotype.vue

```diff
@@ -1,22 +1,123 @@
 <script>
+import { ref } from "vue";
+
 export default {
+  setup(props, { emit }) {
```

Comment:
These code changes are written in the composite API way. Since we are following writing the code using options API, I suggest to please modify these new code changes to follow the options API way.